src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -171,6 +172,14 @@ Nethermind is an Ethereum implementation created with the C# .NET tech stack, li
 
 Nethermind also has [detailed documentation](https://docs.nethermind.io), strong dev support, an online community and 24/7 support available for premium users.
 
+### Ethereumjs {#ethereumjs}
+
+It is written in TypeScript and composed of a number of packages, including core Ethereum primitives represented by the the Block, Transaction, and Merkle-Patricia Trie classes and core client components including an implementation of the Ethereum Virtual Machine (EVM), a blockchain class, and the DevP2P networking stack.
+
+The EthereumJS Execution Client (EthereumJS) has been in active development for some time now. Although it is not production-ready, it already serves a variety of purposes like testing, research (e.g. EIPs) and developer tooling to engage with the protocol.
+
+Learn more about ethereumjs by reading its [documentation](https://ethereumjs.readthedocs.io/en/latest/).
```

Comment:
"ethereumjs" -> "EthereumJS"

This  documentation is outdated and should not be referenced, please refer to the monorepo instead.

Reply:
Fixed. Changed the link to refer to the monorepo, as well.